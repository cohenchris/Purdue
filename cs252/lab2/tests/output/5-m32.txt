0x565b2050
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x565b2008
(nil)
[
	addr: 0008
	size: 4072
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

test: ../my_malloc.c:651: my_free: Assertion `false' failed.
