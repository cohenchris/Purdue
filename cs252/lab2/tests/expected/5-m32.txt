
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

(nil)
[
	addr: 0008
	size: 4072
	left_size: 0
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]
